Framework/CurveFitting/inc/MantidCurveFitting/Functions/ComptonProfile.h

```diff
@@ -65,14 +66,12 @@ class MANTID_CURVEFITTING_DLL ComptonProfile : public API::ParamFunction,
   void buildCaches();
 
   /// Pre-calculate the Y-space values with specified resolution parameters
-  void cacheYSpaceValues(const std::vector<double> &tseconds,
-                         const bool isHistogram,
+  void cacheYSpaceValues(const HistogramData::HistogramX &tseconds,
```

Comment:
This should be `Points` (as in your earlier change), otherwise you cannot remove the  `isHistogram` flag.